src/main/java/tan/ToDo.java

```diff
@@ -0,0 +1,28 @@
+package tan;
+public class ToDo extends Task {
+
+    public ToDo(String description) {
+        super.setTaskName(description);
+    }
+
+    /**
+     * Returns the icon for todo tasks.
+     *
+     * @return Returns the String "T"
+     */
+    public String getIcon() {
```

Comment:
It seems rather unnecessary to have a method to return "T", perhaps there is simpler way of doing this?